src/communication/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.5.3
```

Comment:
I don't think you have the latest in your main, latest is 1.6.0. Please sync your fork and update the PR. Also, for a change of this size, with all the auto generated code that runs behind the scenes updated, I would not recommend a minor version change. I would even recommend to bump the major version and going straight to 2 (2.0.0).

Reply:
@jorge-beauregard, is this a breaking change?

Reply:
I checked that all previous functionality still worked exactly the same way as it used to. Aside from all generated code being revamped, the interaction the final user has is still the same